Corona-Warn-App/src/main/res/layout/fragment_settings_notifications.xml

```diff
@@ -17,16 +17,15 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <include
+        <androidx.appcompat.widget.Toolbar
```

Comment:
Is there a reason why we use `Toolbar` instead of `MaterialToolbar` here?

Reply:
Nope, plain ignorance.